lndr-backend/src/Lndr/Types.hs

```diff
@@ -184,12 +184,12 @@ instance ToJSON DevicePlatform where
    toJSON Ios     = String "ios"
    toJSON Android = String "android"
 
--- This should probably be called 'Push Object' to match the Urban Airship docs
--- here: https://docs.urbanairship.com/api/ua/#push-object
+-- This should match the field names used in the Lambda service
+-- here: hhttps://github.com/blockmason/lndr-notifications
```

Comment:
🐛 Typo here: `hhttps` should be `https`.
Also, instead of the "Lambda service", for consistency and clarity let's say "Notifications API" -- we may end up with more than one Lambda thing later.